runtime/rococo/src/weights/runtime_parachains_configuration.rs

```diff
@@ -148,4 +148,6 @@ impl<T: frame_system::Config> runtime_parachains::configuration::WeightInfo for
 			.saturating_add(T::DbWeight::get().reads(4))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
+
+	fn set_config_with_perbill() -> Weight { todo!() }
```

Comment:
Missing implementation.

(I know you are working on this atm. I'm marking it for convenience.)